webpack/assets/javascripts/react_app/components/Settings/SettingsActions.js

```diff
@@ -5,22 +5,20 @@ import {
   GET_SETTING_FAILURE,
 } from './SettingsConstants';
 
-export const loadSetting = settingName => dispatch => {
+export const loadSetting = settingName => async dispatch => {
   dispatch({ type: GET_SETTING_REQUEST });
-
-  return API.get(`/api/v2/settings/${settingName}`)
-    .then(({ data }) => {
-      dispatch({
-        type: GET_SETTING_SUCCESS,
-        response: data,
-      });
-    })
-    .catch(result => {
-      dispatch({
-        type: GET_SETTING_FAILURE,
-        result,
-      });
+  try {
+    const { data } = await API.get(`/api/v2/settings/${settingName}`);
```

Comment:
Does `API.get` do the `foremanUrl()` call or is this a good chance to fix that?

Reply:
No it doesn't and I don't think this PR should be about it

Reply:
opened https://projects.theforeman.org/issues/27809 to fix this